pop_up_menu.js

```diff
@@ -0,0 +1,272 @@
+// Build function to receive
+function createPopUpMenu(
+  titleText,
+  listItems,
+  imageSrc,
+  descriptionText,
+  linkSource,
+  liveVersion,
+) {
+  const section = document.createElement('section');
+  section.classList.add('pop_up_menu');
+
+  const headerDiv = document.createElement('div');
+  headerDiv.classList.add('pop_menu_header');
+
+  const title = document.createElement('p');
+  title.classList.add('pop_menu_title');
+  title.textContent = titleText;
+  headerDiv.appendChild(title);
+
+  const closeIcon = document.createElement('img');
+  closeIcon.classList.add('pop_menu_cls_icon');
+  closeIcon.alt = 'cancel or close icon';
+  closeIcon.setAttribute('src', './img/cancel_Icon.png');
+  headerDiv.appendChild(closeIcon);
+
+  section.appendChild(headerDiv);
+
+  const list = document.createElement('ul');
+  list.classList.add('pop_menu_list');
+
+  listItems.forEach((itemText) => {
+    const listItem = document.createElement('li');
+    listItem.textContent = itemText;
+    list.appendChild(listItem);
+  });
+
+  section.appendChild(list);
+
+  const contentDiv = document.createElement('div');
+  contentDiv.classList.add('pop_menu_content');
+
+  const image = document.createElement('img');
+  image.classList.add('pop_menu_img');
+  image.src = imageSrc;
+  image.alt = 'work section pop ups';
+  contentDiv.appendChild(image);
+
+  const content1Div = document.createElement('div');
+  content1Div.classList.add('pop_menu_content_1');
+
+  const description = document.createElement('p');
+  description.classList.add('pop_menu_descr');
+  description.textContent = descriptionText;
+  content1Div.appendChild(description);
+
+  const buttonsDiv = document.createElement('div');
+  buttonsDiv.classList.add('pop_menu_buttons');
+
+  const button1 = document.createElement('a');
+  button1.classList.add('pop_menu_btn');
+  button1.setAttribute('href', liveVersion);
+  const span1 = document.createElement('span');
+  span1.textContent = 'See Live';
+  const img1 = document.createElement('img');
+  img1.alt = 'See Live';
+  img1.src = './img/see_live_icon.png';
+  button1.appendChild(span1);
+  button1.appendChild(img1);
+  buttonsDiv.appendChild(button1);
+
+  const button2 = document.createElement('a');
+  button2.classList.add('pop_menu_btn');
+  button2.setAttribute('href', linkSource);
+  const span2 = document.createElement('span');
+  span2.textContent = 'See source';
+
+  const img2 = document.createElement('img');
+  img2.alt = 'github logo';
+  img2.src = './img/social_media_logo/github.png';
+  button2.appendChild(span2);
+  button2.appendChild(img2);
+  buttonsDiv.appendChild(button2);
+
+  content1Div.appendChild(buttonsDiv);
+  contentDiv.appendChild(content1Div);
+  section.appendChild(contentDiv);
+
+  return section;
+}
```

Comment:
- [ ] Well done :clap: you did a nice job but still your pop functionality is not working properly you just implemented this functionality for one card for the other card it's not working make sure to implement this functionality for every card so kindly fix it :+1: 
- [ ] Please stick to the Figma design your pop is not the same as the Figma design you did implement blur background so kindly add blur background as well :slightly_smiling_face: 
- [ ] Please make sure when the user hovers over any of your cards the overlay disappears and the button is shown so kindly add this functionality as well :slightly_smiling_face: check below :point_down:  my video for more information

[screen-capture (13).webm](https://github.com/rychrr/portofolio_page/assets/45226848/1efced2c-0e3a-4390-80fe-4c98e91df780)

**_Note:_** You will implement these changes on both mobile and desktop version